.gitignore

```diff
@@ -8,3 +8,8 @@
 /dataSources.local.xml
 /.idea/
 /target/
+# files
```

Comment:
spoko, że ogarnąłeś, że to trzeba dodać, ale lepiej by było, żeby te pliki były w jakimś konkretnym katalogu tworzone i tutaj dodać cały ten katalog :)